src/js/components/devices/expanded-device.js

```diff
@@ -285,16 +281,6 @@ export const ExpandedDevice = ({
 
   const scrollToMonitor = () => setDetailsTab('monitor');
 
-  const onCreateDeploymentClick = () => navigate(`/deployments?open=true&deviceId=${deviceId}`);
-
-  const actionCallbacks = {
-    onAddDevicesToGroup,
-    onAuthorizationChange,
-    onDeviceDismiss,
-    onRemoveDevicesFromGroup,
-    onPromoteGateway: onMakeGatewayClick,
```

Comment:
How does this end up in the actionCallbacks after your change?

Reply:
Argh I missed that, thanks a lot for catching this!